### ISSUE TITLE
zoom feature re-sizes the view

### ISSUE THREAD

**Issue by nsteinme** [kwikteam/phy-contrib#33]
When I have automatic zoom on, if I set the waveform view to look like this:

![image](https://cloud.githubusercontent.com/assets/7590975/14461033/b4466964-00b8-11e6-845f-4cded127909c.png)

(which looks good to me) then go to the next unit I now see this: 

![image](https://cloud.githubusercontent.com/assets/7590975/14461048/c515767c-00b8-11e6-9774-2ef7bce505c6.png)

(in this case I actually went to the next cluster but then back to the first one, just so it's most clear what the difference is). I would prefer it keep the proportions that I had decided were best for viewing spikes and instead only shifted the "x,y camera position" to be above the new cluster.

**Comment by rossant** [comment]
I can't reproduce this (i.e. the behavior you want is what's implemented). Can you:

* delete .phy
* delete ~/.phy/TemplateGUI/
* update phy and phycontrib to master and python setup.py develop
* try again?

**Comment by nsteinme** [comment]
Ok, just did that, but I am still getting it. I think I figured it out -
the kind of resizing that is not preserved from cluster-to-cluster is the
kind you get when you "right click and drag right" which expands the view
along the horizontal axis. Looks like the kinds you get with alt+arrow keys
and scroll wheel are preserved, and after some playing around I was able to
resize the way I wanted using just those. So if you're satisfied with the
right-click zoom not having this property (or if that's hard for some
reason) then closing the issue is fine with me! Thanks,

Nick
